_rules/device-motion-user-interface-7677a9.md

```diff
@@ -0,0 +1,259 @@
+---
+id: 7677a9
+name: Device motion based functionality can also be operated from the user interface
+rule_type: atomic
+description: |
+  This rule checks that functionality that can be operated by device motion can also be operated by user interface components.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
+
+## Expectation 1
+
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target there exists a set of one or more [instruments][instrument] supporting indistinguishable results of those from the event.
```

Comment:
Same problem as I pointed out for auto-updating. The word "exists" is unspecified. We need to be able to answer "true" or "false" on each expectation individually.

Reply:
Combined expectations